library/init/nat.lean

```diff
@@ -138,9 +138,9 @@ namespace nat
 
   theorem le.rec_on {a : nat} {P : nat → Prop} {b : nat} (H : a ≤ b) (H₁ : P a) (H₂ : ∀ b, a < b → P b) : P b :=
   begin
-    cases H with b' hlt,
+    cases H with b hlt,
```

Comment:
Note that the first `with` parameter is actually irrelevant now, I've renamed it merely for consistency (as in the corresponding `hlean` file). I'll look into generally eliminating irrelevant identifiers next.